pkg/cloud/gcp/actuators/util/gcp_uefi_disk_check_test.go

```diff
@@ -0,0 +1,212 @@
+package util_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	machinev1 "github.com/openshift/api/machine/v1beta1"
+	computeservice "github.com/openshift/machine-api-provider-gcp/pkg/cloud/gcp/actuators/services/compute"
+	"github.com/openshift/machine-api-provider-gcp/pkg/cloud/gcp/actuators/util"
+)
+
+var _ = Describe("IsUEFICompatible", func() {
+	var (
+		computeService computeservice.GCPComputeService
+		providerSpec   *machinev1.GCPMachineProviderSpec
+
+		compatible bool
+		err        error
+	)
```

Comment:
Are these needed still? Are they being shared with the other set of tests at the end?